### PR TITLE
Proposal for Helm graduation

### PR DESCRIPTION
**Pull request by mattfarina**
This proposal follows the process outlined in PR #374. It highlights
how Helm meets the current graduation criteria, following the template,
and links to additional information providing more details on the state
of Helm.

Note, in the SIG App Delivery on March 4th Helm graduation was discussed and the documented reviewed (in draft forms).

### CODE REVIEW

File: proposals/graduation/helm.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+On behalf of the maintainers, we believe Helm is ready for [graduation stage](https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc#graduation-stage) and meets the v1.3 criteria. The Helm [due diligence questions](https://docs.google.com/document/d/1-5ncwLwikeQrAmeVAPuCYlWLYMCw2OmQuouTYCECYFk/edit), following the [guidelines](https://github.com/cncf/toc/blob/master/process/due-diligence-guidelines.md), can be found in a separate Google Docs document.
+
+Helm has seen dramatic growth since joining incubation. For example,
+
+* [At KubeCon/CloudNativeCon US in San Diego Helm was 3rd most popular project](https://www.cncf.io/wp-content/uploads/2020/01/KubeCon_NA_19_Report.pdf).
+* Helm now has more than 1 million downloads in a months time.
+* Installations via Homebrew for MacOS have grown more than 12 times since entering incubation.
+* Helm v3, a major change, was released
```
Comment (technosophos) [inline proposals/graduation/helm.md]: Could we add "and currently comprises XX% of our downloads"

File: proposals/graduation/helm.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Many companies and organizations use Helm. Helm has been downloaded more than a million time in a month, across all the Helm sub-projects there are maintainers from more than 25 companies, the [Helm Hub](https://hub.helm.sh/charts) lists repositories from more than 150 organizations, and many more organizations than that have their people in the Helm issues asking support questions. The [Helm website](https://helm.sh) has long had a list of adopters. More recently the Helm code has added an [adopters file](https://github.com/helm/helm/blob/master/ADOPTERS.md) with a select number of organizations. Organizations are only added when the organization provides permission.
+
+## Incubation Details
+
+_**Project should address each area listed below**_
+
+### * Link to Incubation Due Diligence(DD) Document
+
+Helm entered the CNCF as an incubating project after being part of Kubernetes when it came into the CNCF. Helm was apart of Kubernetes when it graduated. The Helm due diligence at the time is captured in the [incubating proposal](https://github.com/cncf/toc/blob/master/proposals/incubation/helm.adoc).
```
Comment (technosophos) [inline proposals/graduation/helm.md]:
```suggestion
Helm entered the CNCF as an incubating project after being part of Kubernetes when it came into the CNCF. Helm was a part of Kubernetes when it graduated. The Helm due diligence at the time is captured in the [incubating proposal](https://github.com/cncf/toc/blob/master/proposals/incubation/helm.adoc).
```

**Comment by technosophos** [comment]
Other than one small typo, all looks good from the Helm side.

File: proposals/graduation/helm.md (hunk 2 above)
Comment (mattfarina) [reply proposals/graduation/helm.md]: Thanks, fixed.

**Comment by michelleN** [comment]
Thank you. We need a @cncf/toc member to look through due diligence and kick off the public comment period on the mailing list. @amye - can you help us find a TOC member who can do that for us?

**Comment by justincormack** [comment]
Hi, I will look through the due diligence for the ToC.

**Comment by mattfarina** [comment]
@justincormack thanks for picking this up. if you have any questions or need any further detail please feel free to reach out and let me know.

**Comment by justincormack** [comment]
I have opened the public comment period for the next two weeks, please reply in the mailing list thread https://lists.cncf.io/g/cncf-toc/message/4447

**Comment by amye** [comment]
Public comment period is closed, this should now move to a vote.

**Comment by lizrice** [comment]
Given recent discussions about artifact discovery and distribution, it seems worth an explicit note about the TOC’s position on Helm Hub. Helm Hub / monocular provides prior art, and is a valuable part of the Helm ecosystem. The TOC doesn’t see any value in disrupting that work (and in particular we see no value in separating Helm Hub / monocular from the rest of the Helm project at this late stage). We don’t have a firm position yet on what the future of “hubs” look like, but as that work evolves, we’d expect to engage with the Helm community and other projects on the common goal of useful solution(s) that meet the needs of end users who want to discover and obtain artefacts of various types. This is business-as-usual for the TOC and the CNCF family of projects.

**Comment by justincormack** [comment]
Ok we can now go forward to a vote.

**Comment by chris-short** [comment]
I would appreciate greatly appreciate clarity on Helm Hub vs. CNCF Hub vs. Operator Hub vs. INSERT HUB NAME HERE. I feel like we’re sending some conflicting messages

**Comment by caniszczyk** [comment]
+1 Binding votes:
Brendan Burns: https://lists.cncf.io/g/cncf-toc/message/4554
Liz Rice: https://lists.cncf.io/g/cncf-toc/message/4565
Justin Cormack: https://lists.cncf.io/g/cncf-toc/message/4534
Katie Gamanji: https://lists.cncf.io/g/cncf-toc/message/4546
Matt Klein: https://lists.cncf.io/g/cncf-toc/message/4568
Xiang Li: https://lists.cncf.io/g/cncf-toc/message/4569
Alena Prokharchyk: https://lists.cncf.io/g/cncf-toc/message/4583
Michelle Noorali: https://lists.cncf.io/g/cncf-toc/message/4588
Sheng Liang: https://lists.cncf.io/g/cncf-toc/message/4589

**Pull request merged**